### DISCUSSION TITLE
Allow customization of ItemPage's StickySidebar

### DISCUSSION THREAD

**Discussion by maxiruani** [keystonejs/keystone#6693] [Feature requests]
Currently it is only used to show the `Item ID`.
It would be very nice to allow placing a custom view component that receives the whole `data.item` and be able to display read only metadata of the record (like timestamps of creation and status).

**Comment by gwyneplaine** [comment]
@maxiruani  thanks for this, we're presently rethinking how the Admin UI should look. As part of this we're considering removing the sidebar as a feature, in favor of better usage of screen real estate. 

Alongside this however, we're looking to supply new areas of customisation on item pages and other features, to cater to more use-cases like this, stay tuned!
